src/main/java/ch/njol/skript/bukkitutil/EntityUtils.java

```diff
@@ -0,0 +1,120 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.bukkitutil;
+
+import org.bukkit.entity.Ageable;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Piglin;
+import org.bukkit.entity.Zoglin;
+import org.bukkit.entity.Zombie;
+
+import ch.njol.skript.Skript;
+
+/**
+ * Utility class for quick {@link Entity} methods
+ */
+public class EntityUtils {
+	
+	private static final boolean HAS_PIGLINS = Skript.classExists("org.bukkit.entity.Piglin");
+	
+	/**
+	 * Check if an entity is ageable.
+	 * Some entities, such as zombies, do not have an age but can be a baby/adult.
+	 *
+	 * @param entity Entity to check
+	 * @return True if entity is ageable
+	 */
+	public static boolean isAgeable(Entity entity) {
+		if (entity instanceof Ageable || entity instanceof Zombie)
+			return true;
+		else return HAS_PIGLINS && (entity instanceof Piglin || entity instanceof Zoglin);
```

Comment:
Why `else` ?

Reply:
Ah you're right. That was just something IntelliJ did.
I'll remove it!